pep/005-mixins-are-bundles.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Mixins are Bundles"
+number: "005"
+status: "provisional"
+owner: "@carolynvs"
+authors:
+  - "@carolynvs"
+created: "2022-03-03"
+type: "feature"
+---
+
+## Abstract
+
+Let's take advantage of the distribution and security features of bundles for distributing and executing mixins.
+
+## Motivation
+
+Initial Proposal: https://github.com/getporter/porter/discussions/1499
+
+Currently mixins are binaries that are distributed by the porter client by downloading the mixin specified by the user and saving it to the porter home directory (PORTER_HOME).
+When a bundle is built, the mixin binaries located in PORTER_HOME are copied into the bundle's invocation image.
+This is a proposal to distribute the mixin binaries in a bundle instead of distributing the binaries directly.
+
+### Security
+
+___
+
+#### Verify Provenance
+
+> I want to be able to verify that the mixins in a bundle came from their stated source.
+
+Right now, we communicate the name of the mixins included in a bundle, and it is up to the user to decide if the bundle was built securely and actually contains the correct version of the stated mixins.
+A bundle author could use an arbitrary binary named after the mixin mixin, and the end-user couldn't tell the difference.
```

Comment:
Smal nit: double type `mixin mixin`

Reply:
Overall this is a lovely exciting proposal that I assume made things to be done in a more Porter way 🎉 

I am curious on the 2nd point of the Cons mentioned in [this Discussion](https://github.com/getporter/porter/discussions/1499) that said "Running mixins in a bundle is more complex.". Will this affect the experience of the user, how would it be different for users to use mixins in their bundles after these changes?